cmake/BLTPrivateMacros.cmake

```diff
@@ -754,6 +754,24 @@ macro(blt_add_hip_executable)
 
 endmacro(blt_add_hip_executable)
 
+##-----------------------------------------------------------------------------
+## blt_error_if_target_exists(<extension_list> <regex_output_variable>)
```

Comment:
```cmake
## blt_error_if_target_exists(<extension_list> <regex_output_variable>)
...
function(blt_make_regex_from_ext_list ext_list ext_regex)
```
Please make the docs consistent with the function name

```suggestion
## blt_make_regex_from_ext_list(<extension_list> <regex_output_variable>)
```

Reply:
Style note: BLT typically defines this sort of thing as a `macro` with named arguments instead of a `function`